modules/lib/types.nix

```diff
@@ -0,0 +1,22 @@
+{ lib }:
+
+with lib;
+{
+  selectorFunction = mkOptionType {
+    name = "selectorFunction";
+    description =
+      "Function that takes an attribute set and returns a list " +
+      "containing a selection of the values of the input set";
```

Comment:
For consistency the string concatenation `+` should go in the front of this line rather than the end of the previous line.